jsonrpc/debug_endpoint.go

```diff
@@ -248,13 +250,19 @@ func newTracer(config *TraceConfig) (
 		}
 	}
 
-	tracer := structtracer.NewStructTracer(structtracer.Config{
-		EnableMemory:     config.EnableMemory && !config.DisableStructLogs,
-		EnableStack:      !config.DisableStack && !config.DisableStructLogs,
-		EnableStorage:    !config.DisableStorage && !config.DisableStructLogs,
-		EnableReturnData: config.EnableReturnData,
-		EnableStructLogs: !config.DisableStructLogs,
-	})
+	var tracer tracer.Tracer
+
+	if config.Tracer == "callTracer" {
```

Comment:
Maybe we should per-define all available tracers in some constants?

Reply:
I'll move this `callTracer` string to a constant.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/2001/commits/7cb7be10c158113cc224e93b3c8123133040f980